### ISSUE TITLE
[FEATURE REQUEST] Additional check in LethalThings compatibility mode.

### ISSUE THREAD

**Issue by zikeji** [MikeS-MS/MikesTweaks#27]
## **TL;DR:** If the utility belt is disabled in LethalThings don't enable the compatibility mode that is predicated on the utility belt in LethalThings.

**Is your feature request related to a problem? Please describe.**

My group adores LethalThings and would like to keep it enabled, but we're transitioning to MikesTweaks for some of the features.

Your compatibility mode disables the additional hotbar slots when LethalThings is enabled - which is as described, but not desired. This is a different take on #26 in a way however (though a "disable compatibility mode at your own peril" option would solve this, assuming is was specific compatibility modes and not just a blanket disabling all compatibility modes).

In my head, logically, if I disable the utility belt in LethalThings, this should disable the compatibility mode in MikesTweaks allowing the hotbar size tweak to work. This however, is not the case.

**Describe the solution you'd like**

When the mod sees LethalThings installed and enabled, it should further check the LethalThings config (`config/evaisa.lethalthings.cfg`) for the `Items.PouchyBelt` value, which, if false, disables the Utility Belt in LethalThings. If it's explicitly disabled, the compatibility mode should not trigger.

**Describe alternatives you've considered**

Using a different mod to add hotbar slots is a current alternative I'm considering, however HotBarPlus caused some pretty bad desync issues so we're still looking.

**Does any other mod implement this feature already?**

This is a request to change an existing feature, so no.

**Comment by MikeS-MS** [comment]
Check out my comment in #26 

TLDR:
Current compatibility measures are temporary until I can figure out a way to integrate the mods better in a way that won't completely break vanilla players that join your lobby or because of overlapping code insertions.

**Comment by zikeji** [comment]
I read both that explanation and your TL;DR and investigated the LethalThings code a bit. I'm not a modder nor have I used C# anytime recently, so I'm mostly going on speculation, but it looks like LethalThings still loads all the utility belt code, which means the hooks as well, and merely makes it so you can't buy it if it is disabled.

Whether or not this causes an issue in the long run is unknown to me, but it wouldn't make sense to implement my suggestion due to this and as such I'll be closing this request. A temporary override of the compatibility mode like #26 requests is still viable, but I understand not wanting to do so either so I'll leave that to the guy on #26 to petition for.

In any case, thanks for taking the time to respond! Have a great day.
